app/scripts/controller-init/confirmations/transaction-controller-init.ts

```diff
@@ -7,17 +9,17 @@ import {
   TransactionMeta,
   TransactionType,
 } from '@metamask/transaction-controller';
-import SmartTransactionsController from '@metamask/smart-transactions-controller';
-import { SmartTransactionStatuses } from '@metamask/smart-transactions-controller/dist/types';
 import { Hex } from '@metamask/utils';
+import { trace } from '../../../../shared/lib/trace';
 import { getIsSmartTransaction } from '../../../../shared/modules/selectors';
+import { getShieldGatewayConfig } from '../../../../shared/modules/shield';
+import { TransactionMetricsRequest } from '../../../../shared/types/metametrics';
 import {
   SmartTransactionHookMessenger,
-  publishSmartTransactionHook,
-  publishBatchSmartTransactionHook,
+  publishBatchHook,
```

Comment:
Thanks for renaming these, it makes more sense now since we expanded their business logic.